packages/imperative/src/config/cmd/init/init.definition.ts

```diff
@@ -46,20 +46,35 @@ export const initDefinition: ICommandDefinition = {
             description: "Prompt for secure values. Useful for disabling prompting in CI environments.",
             type: "boolean",
             defaultValue: true
+        },
+        {
```

Comment:
I believe that we historically have the "ow" alias for overwrite operations. Would it make sense to add that alias here?